libs/platform/ebpf_platform.h

```diff
@@ -615,10 +615,10 @@ extern "C"
         _In_ const GUID* interface_id,
         _In_opt_ void* provider_binding_context,
         _In_opt_ const ebpf_extension_data_t* provider_data,
-        _In_ const ebpf_extension_dispatch_table_t* provider_dispatch_table,
-        _In_ void* callback_context,
-        _In_ ebpf_provider_client_attach_callback_t client_attach_callback,
-        _In_ ebpf_provider_client_detach_callback_t client_detach_callback);
+        _In_opt_ const ebpf_extension_dispatch_table_t* provider_dispatch_table,
```

Comment:
update line 602-603 to say what null means.  E.g., add "or null if none are exposed".